source/more_sampling_tools.Rmd

```diff
@@ -625,9 +637,8 @@ This is the same error you get for:
 'Gold' & ten_second_trunks
 ```
 
-The point to take away is, that when you are using `&` to combine Boolean
-arrays in Python, remember operator precedence, and, when in doubt, put
-parentheses around the expressions on either side of `&`, as here.
+The take away is, when using `&` to combine boolean arrays in Python,
+put parentheses around the expressions on either side of `&`.
```

Comment:
No need to remind about operator precedence here?

Reply:
It felt like a lot to me, so maybe just edit it down a bit? It feels like a side-note, a warning, but does not warrant extended discussion.

Reply:
Maybe better in a footnote?  I don't know if we can have code cells in footnotes, but we can probably just assert the results, and leave them to copy-paste to see the results.